maplibre/src/render/tile_pipeline.rs

```diff
@@ -84,6 +87,25 @@ impl RenderPipeline for TilePipeline {
                     },
                     count: None,
                 }]])
+            } else if self.raster {
```

Comment:
Not sure yet how we can improve this. This is very specific for a file which tries to keep generic.